clio-server/config/clio-logback.xml.ctmpl

```diff
@@ -4,15 +4,14 @@
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>{{$log_dir}}/clio.log</file>
 
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>{{$log_dir}}/clio.%i.log.gz</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>5</maxIndex>
-        </rollingPolicy>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <!-- roll over daily, inferred by logback from the filename pattern -->
+            <fileNamePattern>{{$log_dir}}/clio-%d{yyyy-MM-dd}.log</fileNamePattern>
```

Comment:
We should probably compress the logs as well, when rolled. I think changing the pattern to `.log.gz` will enable this automatically.